tutorials/mcp/tracing_between_mcp_client_and_server/.env.example

```diff
@@ -0,0 +1,3 @@
+OPENAI_API_KEY='YOUR OPENAI API KEY'
+PHOENIX_COLLECTOR_ENDPOINT='https://app.phoenix.arize.com'
+PHOENIX_CLIENT_HEADERS='api_key=YOUR PHOENIX API KEY'
```

Comment:
hey John just FYI we need to change this to the auth header since this only works for cloud